internal/pkg/deploy/cloudformation/stack/pipeline.go

```diff
@@ -21,12 +21,14 @@ type pipelineStackConfig struct {
 	*deploy.CreatePipelineInput
 }
 
-func newPipelineStackConfig(in *deploy.CreatePipelineInput) *pipelineStackConfig {
+func NewPipelineStackConfig(in *deploy.CreatePipelineInput) *pipelineStackConfig {
 	return &pipelineStackConfig{
 		CreatePipelineInput: in,
 	}
 }
 
+// TODO https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-limits.html
+// Role names limited to 64 chars bc UGH
```

Comment:
Is this comment in the right spot?

Reply:
not exactly but it's the entry point to where the role name gets generated. It'll be removed anyway when I fix it in a future PR.